### ISSUE TITLE
Schema::Config doesn't work with PerlDance::Schema

### ISSUE THREAD

**Issue by racke** [interchange/Perl-Dancer-Conference#78]
```
t/talk.t .. 
ok 1 - An object of class 'PerlDance::Schema' isa 'PerlDance::Schema'
DBIx::Class::Storage::DBI::_warn_undetermined_driver(): Unable to extract a driver name from connect info - this should not have happened. While we will attempt to continue anyway, the results are likely to be underwhelming. Please upgrade DBIC, and if this message does not go away, file a bugreport including the following info:
{
  DBIC_DRIVER => "DBIx::Class::Storage::DBI",
  DBIC_DSN => "PerlDance",
  DBIC_VER => "0.082820",
  DBI_VER => "1.634"
}
```

**Comment by SysPete** [comment]
Looks like this is fixed by a6059fef7c24df4a691c78a4428b0d38ed7aec96 - @racke please confirm.

**Comment by racke** [comment]
Yes.
